i18n/en.toml

```diff
@@ -147,23 +146,23 @@ vehicleTarget = "Vehicle limit: {soc}%"
 
 [main.vehicleStatus]
 minCharge = "minimum charging to {soc}%."
-waitForVehicle = "Ready. Waiting for vehicle."
+waitForVehicle = "Ready. Waiting for vehicle…"
 vehicleTargetReached = "Vehicle limit {soc}% reached."
-charging = "Charging."
-targetChargePlanned = "Target charge planned. Starting {time}."
-targetChargeWaitForVehicle = "Target charge ready. Wait for vehicle."
-targetChargeActive = "Target charge active."
+charging = "Charging…"
+targetChargePlanned = "Charging to target % at {time}."
```

Comment:
This could be confusing, as it says Charging, but at the same time I would see that it is not charging (yet)

Reply:
I've updated the target charging strings. Also removed the `%` since it's redundant information that we currently also don't use non-target charge strings -> Symmetry